### DISCUSSION TITLE
Design Emulator

### DISCUSSION THREAD

**Discussion by getselim-stack** [HASwitchPlate/openHASP#496] [Ideas]
Getting into OpenHASP for the first time using Landon L8s.  Would be really cool if there was an emulator that we could use as a design tool, i.e. a virtual display that I can code and see the results and test without having to design it with trial and error on an actual device.  If such a thing exists I couldn't find it and it's beyond my programming skills to develop.  I am looking to design some fairly advanced front ends and this would really help.

**Comment by mvturnho** [comment]
Check out http://haspdesigner.qrisonline.nl/
Still under development but improving.
There is also a discussion topic. https://github.com/HASwitchPlate/openHASP/discussions/479
